PJ05/src/Exceptions/UserNotFoundException.java

```diff
@@ -2,15 +2,17 @@
 
 import Field.User;
 
-public class UserNotFoundException extends Exception {
+public class UserNotFoundException extends RequestFailedException {
     User user;
```

Comment:
Similarly to previous comment: how can you send a user if the user is not found?